.github/workflows/systemtests-playground.yml

```
@@ -0,0 +1,27 @@
+name: Systemtests Playground
+on:
+  workflow_call:
+    inputs:
+      precice-version:
+        type: string
+        description: 'Version of the Precice library to use (SHA or tag)'
```

😱 (shame! 🔔 shame! 🔔 shame! 🔔) 😅

```suggestion
        description: 'Version of the preCICE library to use (SHA or tag)'
```